commcare_connect/organization/migrations/0013_backfill_organization_profile_from_entity.py

```
@@ -0,0 +1,35 @@
+    Organization = apps.get_model("organization", "Organization")
+    organizations = list(Organization.objects.filter(llo_entity__isnull=False).select_related("llo_entity"))
+    for org in organizations:
+        for field in ENTITY_PROFILE_FIELDS:
+            setattr(org, field, getattr(org.llo_entity, field))
+    Organization.objects.bulk_update(organizations, ENTITY_PROFILE_FIELDS, batch_size=500)
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("organization", "0012_organization_profile_fields"),
+    ]
```

Ruff (0.16.1)

[warning] Mutable default value for class attribute

(RUF012)